dbt/adapters/databricks/__version__.py

```diff
@@ -1 +1 @@
-version: str = "1.8.7"
+version: str = "1.9.0b1"
```

Comment:
shall the version bumping be done in a separated PR?

Reply:
I can if you would prefer.  My reasoning is that this is more of just a miss (not having this from the start in the 1.9.latest branch), and saves another round of integration tests running just to merge the version (which is its own issue that at some point I should address).  When we release 1.9.0, that will be its own version PR.